packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag

```diff
@@ -51,6 +51,10 @@ uniform float uvAnimationScrollXOffset;
 uniform float uvAnimationScrollYOffset;
 uniform float uvAnimationRotationPhase;
 
+#if THREE_VRM_THREE_REVISION >= 157
+  uniform vec3 lightProbe[ 9 ];
+#endif
+
```

Comment:
`uniform vec3 lightProbe[ 9 ]` is still defined in `lights_pars_begin` with a macro `#if defined( USE_LIGHT_PROBES )`, so I think it will cause a problem when we actually want to use light probes.

https://github.com/mrdoob/three.js/blob/r157/src/renderers/shaders/ShaderChunk/lights_pars_begin.glsl.js#L5-L9

Reply:
commit: https://github.com/pixiv/three-vrm/pull/1304/commits/3cf43b95215259c4e395081c6b423f39be554832